### ISSUE TITLE
Cannot install old versions of visualstudio2017buildtools, properly

### ISSUE THREAD

**Issue by brettjacobson** [jberezanski/ChocolateyPackages#73]
The chocolatey package for .NET framework 4.8 (dotnetfx 4.8) breaks cholatey and Pester in interesting ways.

Unfortunately, when trying to install an OLDER version of Visual Studio 2017 Build Tools using this chocolatey package, it always installs the dotnetfx48 package (likely because the version number is not pinned, thus it installs the latest version).  This makes rolling back to install an older vs2017bt version untenable.

**Comment by jberezanski** [comment]
Interesting. I have not noticed any chocolatey breakage on .NET 4.8 and we have a couple dozen machines at work configured with choco, .NET 4.8 and VS 2017. I suggest you submit issues against Choco and Pester.

Pinning the VS dependency version number would not make sense, because:
1) Visual Studio 2017 runs on .NET 4.8 just fine,
2) it would prevent VS installation on machines on which dotnetfx 4.8 is already installed,
3) it would prevent dotnetfx upgrade once the user decided to do so.

When I find time, I will update the VS packages to use a different .NET package, one that installs only the specific minimum version required by VS and does nothing if a newer .NET version is present.

Meanwhile, you can work around this by explicitly installing the specific version of the dotnetfx package prior to installing Visual Studio:
```
cinst -y dotnetfx --version 4.7.2.20180712   # 4.6.2 or later will do
cinst -y visualstudio2017buildtools
```
(choco will not upgrade the dotnetfx package while installing visualstudio2017buildtools
 if the installed package version satisfies the visualstudio2017buildtools dependency requirement)

**Comment by jberezanski** [comment]
As a side note, please be aware that it is not possible* to install older VS 2017 versions using my packages published on chocolatey.org, because the installer downloaded from Microsoft always installs the latest available VS version. Even the earliest available package (15.0.26228.0) will actually install VS 15.9.16 today. (The package version number is only useful for triggering VS update via choco.)

(* at least not without additional work - providing your own older channel and catalog manifests)
